scopes/design/ui/Avatar/styles.module.scss

```diff
@@ -27,6 +27,9 @@
     text-align: center;
     display: inline-block;
   }
+  .tooltip {
+    max-width: 13ch;
```

Comment:
why this?
what happens when it overflows?

Reply:
No need for this, I removed it because I use the ellipsis css from design.